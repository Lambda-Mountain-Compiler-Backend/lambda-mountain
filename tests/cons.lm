
main := (
   print-s (() ())
#   print-s (123 () 456)
);
