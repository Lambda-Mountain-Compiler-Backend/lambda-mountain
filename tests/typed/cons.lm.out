(() (GExpr (: ((App ((: ((Literal 1) Atom)) (: ((Literal 2) Atom)))) Cons))))
